downstream/modules/platform/proc-controller-create-custom-notifications.adoc

```diff
@@ -0,0 +1,156 @@
+You can customize these messages with different criteria by adding your own attributes for the job as needed. 
+Custom notification messages are rendered using Jinja; the same templating engine used by Ansible playbooks.
+
+Messages and message bodies have different types of content, as the following points outline:
+
+* Messages are always just strings, one-liners only. 
+New lines are not supported.
+* Message bodies are either a dictionary or a block of text:
+** The message body for Webhooks and PagerDuty uses dictionary definitions. 
+The default message body for these is `{{ job_metadata }}`, you can either leave that as it is or provide your own dictionary.
+** The message body for email uses a block of text or a multi-line string. The default message body is:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+{{ job_friendly_name }} #{{ job.id }} had status {{ job.status }}, view details at {{ url }} {{ job_metadata }}
+----
++
+You can edit this text leaving `{{ job_metadata }}` in, or drop `{{ job_metadata }}`. 
+Since the body is a block of text, it can be any string you want.
+`{{ job_metadata }}` is rendered as a dictionary containing fields that describe the job being executed. 
+In all cases, {{ job_metadata }} includes the following fields:
+
+*** `id`
+*** `name`
+*** `url`
+*** `created_by`
+*** `started`
+*** `finished`
+*** `status`
+*** `traceback`
++
+[NOTE]
+====
+You cannot query individual fields within `{{ job_metadata }}`. 
+When you use `{{ job_metadata }}` in a notification template, all data is returned.
+====
++
+The resulting dictionary looks like the following:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+{"id": 18,
+ "name": "Project - Space Procedures",
+ "url": "https://host/#/jobs/project/18",
+ "created_by": "admin",
+ "started": "2019-10-26T00:20:45.139356+00:00",
+ "finished": "2019-10-26T00:20:55.769713+00:00",
+ "status": "successful",
+ "traceback": ""
+}
+----
++
+If `{{ job_metadata }}` is rendered in a job, it includes the following additional fields:
++
+*** `inventory`
+*** `project`
+*** `playbook`
+*** `credential`
+*** `limit`
+*** `extra_vars`
+*** `hosts`
++
+The resulting dictionary looks like the following:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+{"id": 12,
+ "name": "JobTemplate - Launch Rockets",
+ "url": "https://host/#/jobs/playbook/12",
+ "created_by": "admin",
+ "started": "2019-10-26T00:02:07.943774+00:00",
+ "finished": null,
+ "status": "running",
+ "traceback": "",
+ "inventory": "Inventory - Fleet",
+ "project": "Project - Space Procedures",
+ "playbook": "launch.yml",
+ "credential": "Credential - Mission Control",
+ "limit": "",
+ "extra_vars": "{}",
+ "hosts": {}
+}
+----
++
+If `{{ job_metadata }}` is rendered in a workflow job, it includes the following additional field:
++
+*** `body` (This enumerates the nodes in the workflow job and includes a description of the job associated with each node)
```

Comment:
Why 3?

Reply:
Because with 2, in the build it was showing indented to the left but it needs to be indented to the right as with the previous bulleted list.